ui/src/app/controls/multi-key-value/multi-key-value.component.html

```diff
@@ -1,18 +1,18 @@
 <div class="group">
     <div *ngFor="let control of form.controls; index as i">
         <ng-container [formGroup]="control">
-                <mat-form-field class="mid-width" appearance="outline">
+                <mat-form-field class="kv-width" appearance="outline">
                     <mat-label><span>Name</span></mat-label>
                     <input [attr.data-cy]="dataCyPrefix+'-name-'+i" matInput formControlName="name">
                 </mat-form-field>
-                <mat-form-field class="mid-width" appearance="outline">
+                <mat-form-field class="kv-width" appearance="outline">
                     <mat-label><span>Value</span></mat-label>
                     <input [attr.data-cy]="dataCyPrefix+'-value-'+i" matInput formControlName="value">
                 </mat-form-field>
+                <button [attr.data-cy]="dataCyPrefix+'-minus-'+i" mat-icon-button (click)="removeEntry(i)">
+                    <mat-icon class="tab-icon material-icons-outlined">remove</mat-icon>
```

Comment:
I wonder if a trash icon would be more explicit than a minus sign. Did you try?

Reply:
Haven't tried a trash icon, but it seems to be interesting too:

![image](https://github.com/redhat-developer/odo/assets/593208/70150909-e86b-4621-881a-216c46081fb4)

I just tried a small variant of this icon, and it looks even more explicit to me:

![image](https://github.com/redhat-developer/odo/assets/593208/384c307d-b0b7-46c8-8465-98343e93288b)

Thoughts?

Reply:
Yes, the second one seems the best one.

Could you also add some margin on the right of the icon (or remove space on the left), so it is closer to the element it is attached to?

You may be able to "move" the icon with such css:
```
position: relative;
left: 4px;
bottom: 4px; // this one to align the icon vertically with the input
```

Reply:
I didn't notice any noteworthy changes in the positioning of this button, but you can take a look at [`4bf895f` (#7084)](https://github.com/redhat-developer/odo/pull/7084/commits/4bf895f272bb23763471ea18affb8ca93c6e6d55)

Reply:
Sorry, I was talking about the icons on the right of the Command / Args inputs (and probably others with the same disposition).

ui/src/app/controls/multi-text/multi-text.component.css:
```
+button.adjust-position {
+    right: 6px;
+    top: 6px;
+}
```

ui/src/app/controls/multi-text/multi-text.component.html:
```
-            <button [attr.data-cy]="dataCyPrefix+'-minus-'+i" mat-icon-button [matTooltip]="deleteLabel" (click)="removeText(i)">
+            <button class="adjust-position" [attr.data-cy]="dataCyPrefix+'-minus-'+i" mat-icon-button [matTooltip]="deleteLabel" (click)="removeText(i)">
```

Reply:
Ah right, I see. I'll make the changes. Thanks!

Reply:
Done in [`3001acb` (#7084)](https://github.com/redhat-developer/odo/pull/7084/commits/3001acb3c7ea61e584deafef375e9998751daa65)